build/generate/featureMatrix.go

```diff
@@ -212,6 +214,10 @@ func matrixData() *FeatureMatrix {
 			DomainModifierTlsa,
 			providers.CanUseTLSA,
 		)
+		setCapability(
```

Comment:
Please keep this list alphabetized. (and if anything else is out of place, also move it)